src/platforms/hubspot/schemas.ts

```diff
@@ -470,6 +470,7 @@ const callPropertiesSchema = z.object({
   hs_timestamp: custom.date().nullable(),
   hs_call_body: z.string().nullable(),
   hs_call_title: z.string().nullable(),
+  hs_call_recording_url: z.string().url().nullable(),
```

Comment:
nit: for downstream calls, we try not to be too specific on the zod validators, so here `z.string().nullable()` would suffice

Reply:
perfect, scoped it down